### DISCUSSION TITLE
Error during migration

### DISCUSSION THREAD

**Discussion by flying-catsailor** [iotaledger/legacy-migration-tool#141] [Help & Support]
Hello everyone I know I'm a bit late. But I have the problem that my trinity wallet cannot be migrated. After importing the kdbx file, the correct amount status is displayed, but when I start the migration, I am shown that the base token cannot be found. If I go into the legacy network explorer and enter my seed, but it is found. What am I doing wrong, or what do I have to do differently. I am grateful for any help

**Comment by vbgweb** [comment]
Hi, same problem here. Hope for some news and solution. Thanks!

**Comment by begonaalvarezd** [comment]
The issue `errorMessage: cannot debit (base tokens: XXX) from YYY: not enough funds` has been mitigated, please try the migration again. We apologize for the troubles :pray:

**Comment by flying-catsailor** [comment]
Thanks a lot. Now it has wordenKind regards  S. WuschkeAm 16.12.2024 16:35 schrieb Begoña Álvarez de la Cruz ***@***.***>:
The issue errorMessage: cannot debit (base tokens: XXX) from YYY: not enough funds has been mitigated, please try the migration again. We apologize for the troubles 🙏

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/iotaledger/legacy-migration-tool/discussions/141#discussioncomment-11582638",
"url": "https://github.com/iotaledger/legacy-migration-tool/discussions/141#discussioncomment-11582638",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
